src/components/tl/mlx5/mcast/tl_mlx5_mcast.h

```diff
@@ -152,6 +152,7 @@ typedef struct ucc_tl_mlx5_mcast_coll_context {
     ucc_rcache_t                  *rcache;
     ucc_tl_mlx5_mcast_ctx_params_t params;
     ucc_base_lib_t                *lib;
+    enum ucc_tl_capabilities       tl_caps;
```

Comment:
Won't this introduce backwards compatibility issues? @Sergei-Lebedev @samnordmann

Reply:
I don't think it creates any backward compatibility issue, because `struct ucc_context` aka `ucc_context_h` is not supposed to be given by the user, but is set by `ucc_context_create`. So, AFAICS, it is not problematic.

However, it is true that `struct ucc_context` belongs to the main API, therefore modifying might be considered undesirable...